docs/code-of-conduct.md

```diff
@@ -1,86 +0,0 @@
-Code of Conduct
-===============
-
-Short Version
--------------
-
-
-**Write the Docs** is dedicated to providing a harassment-free
-experience for everyone, regardless of gender, sexual orientation,
-disability, physical appearance, body size, race, or religion. We do not
-tolerate harassment of event participants in any form. Sexual language
-and imagery is not appropriate for any event venue, including talks.
-Event participants violating these rules may be sanctioned or expelled
-from the event without a refund at the discretion of the event
-organizers.
-
- 
-
-Long Version
-------------
-
-
-### Harassment
-
-Harassment includes offensive verbal comments related to gender, sexual
-orientation, disability, physical appearance, body size, race, religion,
-sexual images in public spaces, deliberate intimidation, stalking,
-following, harassing photography or recording, sustained disruption of
-talks or other events, inappropriate physical contact, and unwelcome
-sexual attention. Participants asked to stop any harassing behavior are
-expected to comply immediately.
-
-### Exhibitors & Sponsors
-
-Exhibitors in the expo hall, sponsor or vendor booths, or similar
-activities are also subject to the anti-harassment policy. In
-particular, exhibitors should not use sexualized images, activities, or
-other material. Booth staff (including volunteers) should not use
-sexualized clothing/uniforms/costumes, or otherwise create a sexualized
-environment.
-
-### Participants
-
-If a participant engages in harassing behavior, the event organizers may
-take any action they deem appropriate, including warning the offender or
-expulsion from the event with no refund. If you are being harassed,
-notice that someone else is being harassed, or have any other concerns,
-please **contact a member of event staff immediately**. Event staff can
-be identified by t-shirts/special badges.
-
-Event staff will be happy to help participants contact hotel/venue
-security or local law enforcement, provide escorts, or otherwise assist
-those experiencing harassment to feel safe for the duration of the
-event. We value your attendance.
-
-### Venue & Social Events
-
-We expect participants to follow these rules at all related venues and
-social events.
-
-------------------------------------------------------------------------
-
-### Staff Contact Information
-
-**Samuel Wright**
-
-* europe@writethedocs.org
-* +420 775-240-730
-
```

Comment:
We also need to be slightly careful with these contact details. For example that Czech number is inactive when I'm not in Prague. So just listing 3 names regardless of conf does not work.

Is there an advantage to having one *primary* number / email as main point of contact?